### PR TITLE
Update ForgeGradle, update Gradle wrapper, add gitignores, add lastPhysicsProblem, add ATs

### PR DESCRIPTION
**Pull request by josephtarbit**
1. Update to FG 4.0.13
2. Update Gradle to 6.8.1
3. Add a bunch of .gitignores
4. Add `lastPhysicsProblem` to `World`
5. Add some ATs to allow support for FAWE and GriefPrevention

### CODE REVIEW

**Comment by IzzelAliz** [comment]
I noticed that after upgrading the package size changed a little bit and it won't start.

**Comment by IzzelAliz** [comment]
Seems you need to add mixin tools as annotation processor.

**Comment by josephtarbit** [comment]
Ahhh. that would explain it. I'll do that now.

File: arclight-common/build.gradle [hunk 1]
```diff
@@ -51,25 +51,32 @@ repositories {
...
+    implementation project(':arclight-api')
+    implementation project(':i18n-config')
 }
 
 remapSpigotJar {
     includes.add('net/minecraft/block/ChestBlock$DoubleInventory')
     includes.add('net/minecraft/tileentity/LecternTileEntity$LecternInventory')
 }
 
+remapSpigotJar.finalizedBy('reobfJar')
```
Comment (IzzelAliz) [inline arclight-common/build.gradle]: This seems not right since it will force compile when setting up spigot.

**Pull request merged**